doc/assets/img/pydicom_blue_light.svg

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Oh, you put my favorite in, too! Where are you going to use it?

Reply:
I suggested to put it in because I like it too :) The blue and yellow looks really sharp because it subtly hints at python, eg:

![image](https://user-images.githubusercontent.com/814322/36345543-62c398e4-13e1-11e8-83d1-cd411a82febb.png)
![image](https://user-images.githubusercontent.com/814322/36345554-b93ef2fe-13e1-11e8-903f-4dcf3a119e8e.png)

and the dual reference with a snake AND colors is just, fantastic, haha. 

We lose that a bit without the yellow, albeit I still like it a lot:

![image](https://user-images.githubusercontent.com/814322/36345549-999722be-13e1-11e8-91e9-95b15ab4b06f.png)

And I think the yellow accent gives us more opportunity to use that as a highlight / secondary color on the docs, for example remember the sphinx gallery:

![screenshot from 2018-02-17 12-55-59](https://user-images.githubusercontent.com/814322/36345570-012c7c94-13e2-11e8-9df8-6f14ee2e5dab.png)

This is all, of course, subjective. But I think that subtle hints / decisions about branding are generally more important than we deem them to be.

Reply:
That said, I like them both a lot. Snakes and python P, you can't go wrong! :snake: + P = :blue_heart:

Reply:
You do know of course that Python has nothing to with snakes :P

Reply:
> the blue and yellow looks really sharp because it subtly hints at python

+1 for this ( +1'd the wrong comment at first, although that one is fine too)

Reply:
That's not what Snakey McSnakey told me! 
![image](https://user-images.githubusercontent.com/814322/36345689-a0d00520-13e4-11e8-84b7-9d529508b467.png)

Reply:
And in case that was not clear - I really like your logos, all of them!